maven/org.eclipse.emf.mwe2.target/org.eclipse.emf.mwe2.target.target

```diff
@@ -11,7 +11,7 @@
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
-<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.20"/>
+<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.29"/>
```

Comment:
Would it make sense to mention the EMF version in the manifests, too?

Reply:
created https://github.com/eclipse/mwe/issues/241